src/main/java/qna/domain/Question.java

```diff
@@ -14,12 +17,11 @@
     @Lob
     private String contents;
     @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "writerId")
+    @JoinColumn(name = "writer_Id")
     private User writer;
     @Column(nullable = false)
     private boolean deleted = false;
-    @OneToMany(mappedBy = "question", fetch = FetchType.LAZY, cascade = CascadeType.ALL)
-    List<Answer> answers = new ArrayList<>();
+    AnswersRelatedQuestion answersRelatedQuestion = new AnswersRelatedQuestion();
```

Comment:
임베디드 타입이 정상적으로 동작하고 있을까요 ?
@Embedded 어노테이션을 깜빡하신 것 같아요 :)

https://velog.io/@conatuseus/JPA-%EC%9E%84%EB%B2%A0%EB%94%94%EB%93%9C-%ED%83%80%EC%9E%85embedded-type-8ak3ygq8wo

```suggestion
    @Embedded
    AnswersRelatedQuestion answersRelatedQuestion = new AnswersRelatedQuestion();
```

Reply:
@Embedded 와 @Embeddable 둘 중 하나는 생략해도 된다고 해서 @Embedded를 생략했습니다!